src/Grpc/orchestrator_service.proto

```diff
@@ -75,6 +75,7 @@ message ExecutionStartedEvent {
     google.protobuf.Timestamp scheduledStartTimestamp = 6;
     TraceContext parentTraceContext = 7;
     google.protobuf.StringValue orchestrationSpanID = 8;
+    map<string, string> tags = 9;
```

Comment:
Protobuf changes need to be made first in `microsoft/durabletask-protobuf` and then synchronized to this repo using the PowerShell script in this directory.

Reply:
fixed